lighthouse-plugin-ad-speed-insights/utils/bidder-patterns.js

```diff
@@ -16,489 +16,490 @@ module.exports = [
   {
     label: 'Prebid JS',
     patterns: [
-      `^http[s]?:\/\/([^.]*.)?prebid\.org\/.*`,
-      '^http[s]?:\/\/acdn\.adnxs\.com/prebid/.*',
+      `^http[s]?://([^.]*.)?prebid[.]org/.*`,
+      '^http[s]?://acdn[.]adnxs[.]com/prebid/.*',
     ],
   },
   {
     label: 'Media.net',
     patterns: [
-      '^http[s]?:\/\/contextual\.media\.net/bidexchange.*',
+      '^http[s]?://contextual[.]media[.]net/bidexchange.*',
     ],
   },
   {
     label: 'AppNexus',
     patterns: [
-      '^http[s]?:\/\/ib\.adnxs\.com\/jpt.*',
+      '^http[s]?://ib[.]adnxs[.]com/jpt.*',
     ],
   },
   {
     label: 'Amazon',
     patterns: [
-      '^http[s]?:\/\/([^.]*.)?aax\.amazon-adsystem\.com\/.*',
+      '^http[s]?://[a-z-_.]*[.]amazon-adsystem[.]com/.*',
```

Comment:
Was the "aax" removed intentionally? Should this be:
'^http[s]?://([a-z-_.]*[.])?aax[.]amazon-adsystem[.]com/.*'

Reply:
Yes, other subdomains are used for scripts